src/zhinst/toolkit/driver/devices/shfqc.py

```diff
@@ -0,0 +1,10 @@
+"""SHFQA Instrument Driver."""
+
+import logging
+
+from zhinst.toolkit.driver.devices import SHFQA, SHFSG
+
+logger = logging.getLogger(__name__)
```

Comment:
`logger` is not called. Would suggest removing it unless it overwrites SHFQA and SHFSG module loggers.